internal/providers/terraform/aws/dynamodb_table.go

```diff
@@ -40,13 +26,13 @@ func NewDynamoDBTable(d *schema.ResourceData, u *schema.UsageData) *schema.Resou
 
 	args := &aws.DynamoDbTableArguments{
 		Address:        d.Address,
-		Region:         region,
-		BillingMode:    billingMode,
-		WriteCapacity:  writeCapacity,
-		ReadCapacity:   readCapacity,
+		Region:         d.Get("region").String(),
+		BillingMode:    d.Get("billing_mode").String(),
+		WriteCapacity:  intPtr(d.Get("write_capacity").Int()),
+		ReadCapacity:   intPtr(d.Get("read_capacity").Int()),
 		ReplicaRegions: replicaRegions,
 	}
-	args.PopulateUsage(u)
+	keysToSkipSync := []string{"region", "billing_mode", "write_capacity", "read_capacity", "replica_regions"}
```

Comment:
I want to let terraform say "Don't generate the example usage input for these keys since I can pull the values from HCL".  But this is pretty flakey so I'm open to ideas.

Reply:
Could we use the same `infracost:` tags as you suggested below? Something like `infracost:"excludeSync"`?